core/cmd/client.go

```diff
@@ -98,6 +98,10 @@ type ChainlinkAppFactory struct{}
 
 // NewApplication returns a new instance of the node with the given config.
 func (n ChainlinkAppFactory) NewApplication(ctx context.Context, cfg config.GeneralConfig, appLggr logger.Logger, db *sqlx.DB) (app chainlink.Application, err error) {
+	// Avoid double initializations.
+	initOnce.Do(func() {
+		initPrometheus(cfg)
+	})
```

Comment:
Do tests never use this? Otherwise they will race set it and could see confusing results, so we might want to consider erroring out from the test config setup code or something like that.

Reply:
Doesn't look like any parallel tests are using this, I only sequential ones :thinking: 
I also tried running -race multiple time jic but didn't run into races